src/Components/TodoInput.jsx

```diff
@@ -0,0 +1,12 @@
+const inputStyle = {
+  fontSize: '15px',
+};
+
+const TodoInput = ({ todo, onClick, onChange }) => (
+  <>
+    <input type="text" style={inputStyle} value={todo} onChange={onChange} />
+    <button type="button" onClick={() => onClick(todo)}>추가</button>
```

Comment:
여기서 onClick 함수에 `todo`를 전달하더라도 사용하고 있지 않은 것 같습니다. 따라서 다음과 같이 변경하는게 좋을 것 같아요.
```suggestion
    <button type="button" onClick={onClick}>추가</button>
```